tests/conftest.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+from fastapi.testclient import TestClient
+
+from src.dependencies import Session, create_engine, get_session
+from src.main import app
+from src.models import SQLModel
+
+
+@pytest.fixture(scope="function")
+def db_engine(tmp_path):
+    db_path = tmp_path / "test.db"
```

Comment:
What about using an in-memory db? Faster and no side effects (no file to clean up)